t/perf/p7519-fsmonitor.sh

```diff
@@ -135,10 +136,16 @@ test_expect_success "one time repo setup" '
 
 setup_for_fsmonitor() {
 	# set INTEGRATION_SCRIPT depending on the environment
-	if test -n "$INTEGRATION_PATH"
+	if test -n "$USE_FSMONITOR_DAEMON"
 	then
+		git config core.useBuiltinFSMonitor true &&
+		INTEGRATION_SCRIPT=false
+	elif test -n "$INTEGRATION_PATH"
+	then
+		git config --unset core.useBuiltinFSMonitor &&
```

Comment:
aarrgghh!!

we need to guard these "unsets" in case the variable is not set.

```
$ git config --unset core.foo
$ echo $?
5
$
```

Reply:
Of course... ;-)

Or we do The Right Thing and set `core.useBuiltinFSMonitor=false`. That is the correct thing because we would want to override the system/global setting, if there is any.